packages/traceability-schemas/scripts/schemas-to-vocab.js

```diff
@@ -129,9 +121,49 @@ const separateSchemas = (schemaList) => {
   return { credentials, vocab };
 };
 
+const buildWorkflowSection = () => {
+  const images = {
+    "Business Card Workflow": "resources/wf_businesscard.svg"
```

Comment:
why use mermaid, if you we don't render it?

https://mermaid.js.org/config/usage.html

^ lets do this, for the `mermaid` block... and maybe add an example that uses an image, so we cover both cases.